pkg/controller/vm_controller.go

```diff
@@ -446,67 +446,51 @@ func (r *VMReconciler) buildVMPod(ctx context.Context, vm *virtv1alpha1.VirtualM
 				Args:            []string{volumeMount.MountPath + "/rootfs.raw", strconv.FormatInt(volume.ContainerRootfs.Size.Value(), 10)},
 				VolumeMounts:    []corev1.VolumeMount{volumeMount},
 			})
-		case volume.PersistentVolumeClaim != nil:
+		case volume.PersistentVolumeClaim != nil, volume.DataVolume != nil:
```

Comment:
Can we remove DataVolume source?

Reply:
if we remove DataVolume source and use PVC directly, user have to know the the related PVC of a DataVolume. It's better to keep this logic in Virtink to reduce the burden of user.